eng/jacoco-test-coverage/pom.xml

```diff
@@ -188,14 +188,11 @@
         <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
         <executions>
           <execution>
-            <phase>prepare-package</phase>
+            <id>report-aggregate</id>
+            <phase>verify</phase>
             <goals>
               <goal>report-aggregate</goal>
             </goals>
-            <configuration>
-              <!-- Sets the output directory for the code coverage report. -->
-              <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
```

Comment:
This will require a corresponding update to [index.html](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/code-quality-reports/src/main/resources/index.html#L520)